src/adapter.d.ts

```diff
@@ -99,6 +99,21 @@ export declare interface InvocationInfo {
   requestId?:string;
 }
 
+/**
+ * Logger
+ */
+export declare interface Logger {
+  log(args): void;
```

Comment:
is log really supported?

Reply:
Just reflecting what you have build here: https://github.com/adobe/helix-universal/blob/main/src/utils.js#L113 and there is `log` in the list